templates/admin/user/edit.tmpl

```diff
@@ -151,7 +151,7 @@
 
 				<div class="field">
 					<button class="ui green button">{{.locale.Tr "admin.users.update_profile"}}</button>
-					<div class="ui red button show-modal" data-modal="#delete-user-modal" data-url="{{$.Link}}/delete" data-id="{{.User.ID}}">{{.locale.Tr "admin.users.delete_account"}}</div>
+					<div class="ui red button show-modal" data-modal="#delete-user-modal">{{.locale.Tr "admin.users.delete_account"}}</div>
```

Comment:
Two things in my mind:

1. Here the `data-id="{{.User.ID}}"` is removed, is it correct? IIRC the JS code will fill some `data-xxx` attributes into the form fields.
2. In #18064, the button's CSS class name was changed from `delete-button` to `show-modal`, I am not sure what are the differences and concerns. Now the dialog layout is changed again, could @zeripath take a look at them?

Reply:
I will re-test, but you may be right about 1.
About 2, `delete-button` is how we link it to the generic JS version afaik.

Reply:
`show-modal` is also a generic JS version .... indeed, the code for `show-modal` and `delete-button` are quite similar, that's why I am confused about they two.

Reply:
Okay, I've ran some tests locally again.

The reason `data-id` is no longer needed is because this page has the user ID in the URL, which is what is used when deleting.

At the very least, I was able to:
1. Attempt to delete a user and correctly get back "You can't, they still have repos", and then 
2. Select the purge option, which deleted them successfully.

Reply:
Interesting, it seems to happen to work 😀 

If @zeripath doesn't have more questions, I will vote L-G-T-M later.